app/models/notifier.rb

```diff
@@ -13,8 +12,8 @@ def signup_confirm(user, token)
                      :display_name => user.display_name,
                      :confirm_string => token.token)
 
-      mail :to => user.email,
-           :subject => I18n.t("notifier.signup_confirm.subject")
+      compose_mail user,
```

Comment:
Can we make the first argument part of the hash please - I'd suggest as `:recipient` so that it's a bit more obvious at the call site what that argument is?